content/en/docs/tasks/security/authorization/authz-ingress/index.md

```diff
@@ -12,7 +12,7 @@ using an authorization policy.
 
 An Istio authorization policy supports IP-based allow lists or deny lists as well as
 the attribute-based allow lists or deny lists previously provided by Mixer policy.
-The Mixer policy is deprecated in 1.5 and not recommended for production use.
+The Mixer policy is deprecated in *5 and not recommended for production use.
```

Comment:
What is `*5`? Maybe:
```suggestion
Mixer policy has been deprecated since Istio 1.5 and is not recommended for production use.
```
Or is not even available in 1.8? If so, Maybe remove the sentence completely?

Reply:
That was a search/replace error.  I'll try to find out if Mixer is gone or not.

Reply:
mixer is gone, so i'll remove it.